locals.tf

```diff
@@ -42,6 +42,7 @@ locals {
             volume_type                     = var.default_nodepool_os_disk_type
             volume_size                     = var.default_nodepool_os_disk_size
             iops                            = var.default_nodepool_os_disk_iops
+            encrypted   = var.enable_ebs_encryption
```

Comment:
New element needs to be aligned with the old code or `terraform fmt` needs to be run to clean up the tf files in this repo.

Reply:
Acknowledged. Formatting corrected.